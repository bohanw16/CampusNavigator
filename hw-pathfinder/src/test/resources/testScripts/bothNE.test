# find path when both parent and child do not exist

CreateGraph g

FindPath g n1 n2