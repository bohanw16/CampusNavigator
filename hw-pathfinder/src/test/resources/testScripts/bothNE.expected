# find path when both parent and child do not exist

created graph g

unknown: n1
unknown: n2